ui/src/app/core/settings.service.ts

```diff
@@ -85,11 +106,22 @@ export class SettingsService {
     return this.localStorage.setItem('settings', JSON.stringify(this.currentSettings));
   }
 
-  private createEmptySettingsForProject(projectId: string): ProjectSettings {
+  createEmptySettingsForProject(projectId: string): ProjectSettings {
+    const capabilities = this.capabilitiesService.getCapabilitiesSynchronous();
+    let filters: Filter[] = [];
+    if (capabilities.queryExtensions) {
```

Comment:
Am I reading this correctly in that any queryExtension which is for a boolean field will automatically be applied as `= True`? Doesn't matter right now since we only have the one, but I'd put a comment in `common.ts` around `queryExtensionsDataTypes` indicating that any new fields added there which are of the type `Boolean` will automatically be applied. We may have to just special case adding `hideArchived` here if we ever add any boolean queryExtensions in the future which we do not want to be auto applied.

Reply:
I've added a comment; please let me know if you don't think it's clear enough.